devshell.nix

```diff
@@ -6,10 +6,30 @@ let
       pylsp-mypy
       python-lsp-black
       pyls-isort>=0.2.2
+      flake8
+      flake8-isort
+      flake8-black
     '';
   };
 in
 {
   devshell.packages = [ pythonEnv ];
-  env = [{ name = "MYPY_CACHE_DIR"; eval = "$PRJ_DATA_DIR/mypy_cache"; }];
+  env = [
```

Comment:
Can you uh, explain the sorcery contents of this file?

Reply:
Just ignore them

Reply:
adou :v